dist/logs/logzioLogsFunction/core.js

```diff
@@ -1,60 +1,186 @@
-const logger = require('logzio-nodejs');
-const DataParser = require('./data-parser');
 
-function getCallBackFunction(context) {
-  return function callback(err, bulk) {
-    if (err) {
-      context.err(`logzio-logger error: ${err}`, err);
-      context.bindings.outputBlob = bulk;
+const { ContainerClient } = require('@azure/storage-blob');
+const logger = require('logzio-nodejs');
+const fs = require('fs');
+const util = require('util');
+const appendFileAsync = util.promisify(fs.appendFile);
+const containerName = 'logziologsbackupcontainer';
+const updateTimestamp = log => {
+    if (log.time) {
+        return {
+            ...log,
+            '@timestamp': log.time,
+        };
     }
-    context.done();
-  };
+    return log;
+};
+const updateFolderSize = () => {
+    const stats = fs.statSync(fileName);
+    const fileSizeInKb = stats.size / 1000;
+    folderSize += fileSizeInKb;
+}
+const arrayIsEmpty = array => {
+    return Array.isArray(array) && array.length == 0;
+};
+const getDate = () => {
```

Comment:
I think you won't need it if you'll use `+new Date()` which returns the date in epoch millis, and because you're not iterating too many times it will be unique for every usage IMO

Reply:
and if you need the date check out the methods of `Date`, seems like what you're doing is pretty similliar to `new Date().toISOString()`